package.json

```diff
@@ -12,5 +12,9 @@
   "devDependencies": {
     "lerna": "^6.4.1"
   },
+  "scripts": {
+    "start": "yarn workspaces foreach -vpti run start",
+    "start:dev": "yarn workspaces foreach -vpti run start:dev"
```

Comment:
Can you verify that Lerna's method for running parallel commands isn't an improvement over this? From what I know - you can run two commands with different names using Lerna, which may be better than having to make a start:dev script in RN that just points right back to start. I'd rather keep the scripts in the packages completely independent of the monorepo and let this package.json handle picking which scripts to run.

Reply:
My understanding of Lerna is that it is a higher abstraction over workspaces - so it may provide more functionality and a better way to accomplish this task.

Reply:
Alternatively we could probably just do this with concurrently. I'm not sure what the best practice is.